src/builder/Accounts.sol

```diff
@@ -12,6 +12,7 @@ library Accounts {
         uint256 chainId;
         QuarkState[] quarkStates;
         AssetPositions[] assetPositionsList;
+        CometPosition[] cometPositions;
```

Comment:
This structure is a subset of the `cometPositions` data that is already being generated by Portfolios. It's not necessarily the optimal way to structure this data, but I think we're aiming for doing whatever is simplest for the existing client.